File: platforms/kotlin/examples/geometry/mesh/clear_indices.kt
import org.fragmentcolor.*

val mesh = Mesh()
mesh.addVertices(listOf(Vertex(listOf(-0.5f, -0.5f)), Vertex(listOf(0.5f, -0.5f)), Vertex(listOf(0.0f, 0.5f))))
mesh.setIndices(listOf(0u, 1u, 2u))
mesh.clearIndices(); // back to auto-derived dedup

File: platforms/swift/examples/geometry/mesh/clear_indices.swift
import FragmentColor

let mesh = Mesh()
try mesh.addVertices([
    try Vertex([-0.5, -0.5]),
    try Vertex([ 0.5, -0.5]),
    try Vertex([ 0.0,  0.5]),
])
try mesh.setIndices([0, 1, 2])
mesh.clearIndices(); // back to auto-derived dedup